python/alibiexplainer/alibiexplainer/explainer.py

```diff
@@ -0,0 +1,72 @@
+import kfserving
+from enum import Enum
+from typing import List, Any
+import numpy as np
+import kfserving.protocols.seldon_http as seldon
+from kfserving.protocols.seldon_http import SeldonRequestHandler
+import requests
+import json
+import logging
+from alibiexplainer.anchor_tabular import AnchorTabular
+
+logging.basicConfig(level=kfserving.server.KFSERVER_LOGLEVEL)
+
+
+class ExplainerMethod(Enum):
+    anchor_tabular = "anchor_tabular"
+
+
+class AlibiExplainer(kfserving.KFExplainer):
+    def __init__(self,
+                 model_url: str,
+                 protocol: str,
+                 method: ExplainerMethod,
+                 training_data_url: str = None):
+        super().__init__()
+        self.model_url = model_url
+        self.protocol = protocol
+        self.method = method
+        self.training_data_uri = training_data_url
+        self.training_data = None
+        if self.method is ExplainerMethod.anchor_tabular:
+            self.explainer = AnchorTabular(self._predict_fn)
+        else:
+            raise NotImplementedError
+
+    def load(self):
```

Comment:
We need some sort of "loading" I think, to prevent repeat calls.

Reply:
I think we can just use `self.ready` as tornado would be single threaded